docs/gadgets/profile/tcprtt.md

```diff
@@ -0,0 +1,240 @@
+
+The histogram considers only for the TCP connections that have been already
+established, so it does not take into account the connection phase (3-way TCP
+Handshake). If it is what you are looking for, please check the latency
+information the trace tcpconnect gadget provides. See further information
+[here](../trace/tcpconnect.md#calculating-the-latency-of-a-connection).
+
+By default, the profile tcprtt gadget generates one single histogram per host,
+and one per node in case of Kubernetes. However, it also provides multiple ways
+to analyze specific connections. For instance, we can generate multiple
+histograms separated by local IP addresses to explore all the connections
+generated by each local address. And the same can be done for remote IP
+addresses. In addition, it is also possible to filter by a specific local and/or
+remote address to isolate the analysis.
+
+### On Kubernetes
+
+First of all, let's start the gadget on a terminal:
+
+```bash
+kubectl gadget profile tcprtt
+```
+
+In another terminal, create a server using nginx:
+
+```bash
+kubectl create service nodeport nginx --tcp=80:80
+kubectl create deployment nginx --image=nginx
+```
+
+And then, create a pod to generate some traffic with the server:
+
+```bash
+$ kubectl run -ti --privileged --image wbitt/network-multitool myclientpod -- bash
+# curl nginx
+# curl nginx
+```
+
+If we move back to the first terminal and stop the gadget, it will generate the histograms:
+
+```bash
+All Addresses = ****** [AVG 1211.066824]
+        µs               : count    distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 0        |                                        |
+         8 -> 15         : 114      |***********                             |
+        16 -> 31         : 397      |****************************************|
+        32 -> 63         : 182      |******************                      |
+        64 -> 127        : 49       |****                                    |
+       128 -> 255        : 48       |****                                    |
+       256 -> 511        : 107      |**********                              |
+       512 -> 1023       : 108      |**********                              |
+      1024 -> 2047       : 86       |********                                |
+      2048 -> 4095       : 31       |***                                     |
+      4096 -> 8191       : 111      |***********                             |
+      8192 -> 16383      : 28       |**                                      |
+     16384 -> 32767      : 11       |*                                       |
+```
+
+Take into account that the generated histogram didn't just consider the TCP
+connections we established but also any other TCP connection for each node.
```

Comment:
"Take into account that the generated histogram considers all TCP connections for each node, not only the ones we established" ?